Frameworks/include/MockClass.h

```diff
@@ -503,11 +527,20 @@ VERIFY_ARE_EQUAL(1, callCount);
 #if !defined CUSTOM_MOCK_CALLBACK_VALIDATOR
 namespace Test {
 namespace Mock {
+#ifdef VERIFY_IS_NOT_NULL
 template <typename TFunctor>
 void ValidateCallback(const TFunctor& functor, const wchar_t* name) {
     WEX::TestExecution::SetVerifyOutput verifySettings(WEX::TestExecution::VerifyOutputSettings::LogOnlyFailures);
     VERIFY_IS_NOT_NULL(functor, WEX::Common::String().Format(L"The callback for %s has not been set!", name));
 }
+#elif defined(ASSERT_NE_MSG)
+template <typename TFunctor>
+void ValidateCallback(const TFunctor& functor, const wchar_t* name) {
+	ASSERT_NE_MSG(functor, nullptr, "The callback for %s has not been set!", name);
```

Comment:
>	 [](start = 0, length = 1)

no tabs! please make sure you are clang formatting all code before sending a pr.